lib/scss/theme.scss

```diff
@@ -2,26 +2,16 @@
 $text-muted-color: #606060;
 $border-color: #f0f0f0;
 
-
-// Font
-$font-size-base: 18px;
-$font-family: 'Source Sans Pro', sans-serif;
-$font-weight-bold: 600;
-
-
 // Headings
-$h1-font-size: ($font-size-base - 1) * 1.65;
-$h2-font-size: ($font-size-base - 1) * 1.4;
 $headings-color: #000;
-$headings-font-weight: $font-weight-bold;
-
+$headings-alt-color: #e6522c;
+$headings-font-weight: 500;
```

Comment:
@kaankabalak Font weight 500 is not included when we call the the font files in `index.erb`
`<link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600" rel="stylesheet">` 

I think the 500 weight would not work unless we include it.

Reply:
@rushenn  i just pulled @kaankabalak changes from minio/gluegun. In the index.erb it will now be    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
So we are ok.